jekyll/userguide.md

```diff
@@ -49,6 +57,16 @@ We implement a multi-step bias-correct procedure to ensure forward-looking cyclo
 
 Currently, all climate models are run only under a High Emissions scenario, so the cyclone exposure values presented for the Strong Mitigation and Middle of the Road scenarios are identical to those in the High Emissions scenario. Simulations were also only run out to 2050 due to their substantial computational cost. Therefore exposure values for 2051-2100 represent the mean over the 2030-2050 period. We plan to expand our offering to all three scenarios and beyond 2050 in a future product update.
 
+### Water Stress Score
+
+We model current and forward-looking water stress scores using the World Resource Institute’s Aqueduct model. The state-of-the-art Aqueduct model has been extensively used by researchers in academia and industry to assess portfolio water risk.
+
+The presented water stress score is the ratio of water withdrawal to renewable water availability. Water withdrawal incorporates a diverse set of global data sources, including population, livestock, agriculture, irrigation, and other socioeconomic factors. These withdrawals encompass domestic, industrial, irrigation, and livestock uses. Renewable water availability, which includes surface and groundwater pools, is calculated using a global hydrological model that incorporates meteorological and landscape characteristics.
```

Comment:
The sentence `These withdrawals encompass domestic, industrial, irrigation, and livestock uses.` seems to directly duplicate a portion of the prior sentence.

Reply:
I think this is fine: made a minor tweak to: Water withdrawal incorporates a diverse set of global data sources, including: population, livestock, agriculture, irrigation, and other socioeconomic factors. They encompass domestic, industrial, irrigation, and livestock uses.